lib/rufo/file_finder.rb

```diff
@@ -28,9 +41,17 @@ def each
   attr_reader :files_or_dirs
 
   def all_rb_files(file_or_dir)
-    Dir.glob(
-      File.join(file_or_dir, "**", "{*.rb,Gemfile,*.gemspec,#{RAKEFILES},*.rake}"),
-      File::FNM_EXTGLOB
-    ).select(&File.method(:file?))
+    files = []
+    Find.find(file_or_dir) do |path|
+      basename = File.basename(path)
+      if File.directory?(path)
+        Find.prune if basename == "vendor"
```

Comment:
Should we pull this hard-coded into a constant like `EXCLUDED_DIRS` and utilize it here?

```suggestion
        Find.prune if EXCLUDED_DIRS.include?(basename)
```

Reply:
Good idea